salt/master.py

```diff
@@ -1269,9 +1269,12 @@ def _file_recv(self, load):
             return {}
         load.pop('tok')
 
+        # Join path
+        sep_path = os.sep.join(load['path'])
```

Comment:
Was the removal of the `*` to unpack the list an intentional choice here? IIRC, there was a reason we needed this here.